pkg/autocomplete/zsh.go

```diff
@@ -52,33 +53,62 @@ _cli_zsh_autocomplete() {
 
 compdef _cli_zsh_autocomplete envd`
 
+var zshConfig = `
+# envd zsh-completion
+[ -f ~/.envd.zsh ] && source ~/.envd.zsh
```

Comment:
Please use the path ~/.config/envd/

Reply:
I am wondering what will happen if we deleted envd?

Reply:
If the `.envd.zsh` has been deleted, the condition `[ -f ~/.envd.ssh]` will be false and nothing will happen.

BTW, if the envd has been installed more than once and always use the third method(`~/.envd.zsh` and `~/.zshrc`) on a machine, then the `~/.zshrc` will have multiple same lines of `[ -f ~/.envd.zsh ] && source ~/.envd.zsh`. Fortunately, it will not affect the user, not a big deal.

Reply:
Gotcha, then I think it should be in `~/.config/envd`

Reply:
> Please use the path ~/.config/envd/ 

plus, should we change script's name to `.envd.zsh`? Or just keep the old name `_envd`.

Reply:
Maybe use envd.zsh.

There is no need to hide it in the file system.

Reply:
I changed the code to `~/.config/envd/` directory, think it as resolved.